src/components/scale-button.js

```diff
@@ -63,85 +72,71 @@ AFRAME.registerComponent("scale-button", {
     this.objectMatrix = new THREE.Matrix4();
     this.dragVector = new THREE.Vector3();
     this.currentObjectScale = new THREE.Vector3();
-    NAF.utils.getNetworkedEntity(this.el).then(networkedEl => {
-      this.networkedEl = networkedEl;
-      this.objectToScale = networkedEl.object3D;
-    });
-    const camPosition = new THREE.Vector3();
-    const objectPosition = new THREE.Vector3();
-    const objectToCam = new THREE.Vector3();
-    this.startScaling = e => {
-      if (this.isScaling || !this.objectToScale) {
-        return;
-      }
+  }
 
-      if (!(NAF.utils.isMine(this.networkedEl) || NAF.utils.takeOwnership(this.networkedEl))) {
-        return;
-      }
+  startScaling(object3D) {
```

Comment:
This is temporary transition code so it doesn't really matter much, but we might try to name this parameter a bit better. Maybe cursorObj? interactor?